docs/hacking-tips.md

```diff
@@ -30,7 +30,7 @@ Getting errors like these?
 Use [`--trace-warnings`](https://medium.com/@jasnell/introducing-process-warnings-in-node-v6-3096700537ee) to get actual stack traces.
 
 ```sh
-node --trace-warnings lighthouse-cli http://example.com
```

Comment:
Would it be possible to create a CLI alias just for this, without doing any package.json export BS?

Reply:
you can add lighthouse-core/ to your local gitignore and do whatever you want in there